src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/EnumWithFlagsAttribute.cs

```diff
@@ -165,8 +165,6 @@ private static bool ShouldBeFlags(IList<ulong> enumValues)
 
         private static ulong GetMissingBitsInBinaryForm(IList<ulong> values)
         {
-            Debug.Assert(values != null);
```

Comment:
💡 You can use `RoslynDebug.Assert` for cases like this if you want. Or you can just go with the non-null parameter type as good assurance.

Reply:
I think the non-nullable parameter type should be good enough for most cases. I believe `RoslynDebug.Assert` is just a workaround to help the compiler when it reports false positives. Let me know if you believe otherwise.